explorer/interpreter/value.cpp

```diff
@@ -33,6 +38,19 @@ auto StructValue::FindField(std::string_view name) const
   return std::nullopt;
 }
 
+static auto FindClassField(Nonnull<const NominalClassValue*> object,
+                           std::string_view name)
+    -> std::optional<Nonnull<const Value*>> {
+  if (auto field = cast<StructValue>(object->inits()).FindField(name);
+      field.has_value()) {
```

Comment:
```suggestion
  if (auto field = cast<StructValue>(object->inits()).FindField(name)) {
```
The explicit field.has_value() isn't necessary here since the `if` will handle optional, and I think we generally aren't adding it.

Reply:
Done